src/passes/analysis/verilog.cpp

```diff
@@ -1411,10 +1411,10 @@ Passes::Verilog::compileModuleBody(
           throw std::runtime_error("Unexpected compile_guard type: " + type);
         }
       }
-      this->getSymbolTable()->setInstanceName(
-          definition->getModule()->getName(),
-          instance.second->getInstname(),
-          instance_name);
+      // this->getSymbolTable()->setInstanceName(
```

Comment:
Should remove

Reply:
I should really insert a `"RENAME"` here but we'll have to deal with that in finalization later. Removing for now and keeping a TODO comment.